message-me/app/models/message.rb

```diff
@@ -0,0 +1,4 @@
+class Message < ApplicationRecord
+  belongs_to :user
```

Comment:
If here have a `belongs_to`, make sure the corresponding model has a `has_xx` as well